spec/support/models/animals.rb

```diff
@@ -0,0 +1,33 @@
+require 'support/active_record_schema'
```

Comment:
These requires I thought would make it obvious where `ar_schema` comes from, so it's easier to follow the logic in an integration spec from
`require 'support/models/<model>`
to
`require 'support/active_record_schema`
in a few clicks and get a grasp of what is happening.